argo-lite/src/components/ProjectDetailDialog.jsx

```diff
@@ -78,11 +78,12 @@ class ProjectDetailDialog extends React.Component {
                           <Button
                             className={classnames([Classes.BUTTON])}
                             onClick={() => {
-                              // TODO: Open a confirmation dialog
-                              requestDelete(snapshotPath);
-                              appState.project.currentProject.snapshotPaths = appState.project.currentProject.snapshotPaths.filter(
-                                path => path !== snapshotPath
-                              );
+                            if (confirm("This action will delete the snapshot. Would you like to proceed?")) {
```

Comment:
Actually this file is for the electron version of Argo, not relevant for Argo Lite.